db/seeds.rb

```diff
@@ -136,14 +168,33 @@
                                     browser_url: nil}],
                    data_type: 'string', is_ontology_based: true, is_array_based: true, big_query_id_column: 'disease',
                    big_query_name_column: 'disease__ontology_label', convention_name: 'Alexandria Metadata Convention',
-                   convention_version: '1.1.3')
-SearchFacet.create(name: 'Organism Age', identifier: 'organism_age', big_query_id_column: 'organism_age', big_query_name_column: 'organism_age',
+                   convention_version: '2.2.0')
+SearchFacet.create!(name: 'Organism Age', identifier: 'organism_age', big_query_id_column: 'organism_age', big_query_name_column: 'organism_age',
                    big_query_conversion_column: 'organism_age__seconds', is_ontology_based: false, data_type: 'number',
-                   is_array_based: false, convention_name: 'Alexandria Metadata Convention', convention_version: '1.1.3',
+                   is_array_based: false, convention_name: 'Alexandria Metadata Convention', convention_version: '2.2.0',
                    unit: 'years')
 
-BrandingGroup.create(name: 'Test Brand', user_id: api_user.id, font_family: 'Helvetica Neue, sans-serif', background_color: '#FFFFFF')
+BrandingGroup.create!(name: 'Test Brand', user_id: api_user.id, font_family: 'Helvetica Neue, sans-serif', background_color: '#FFFFFF')
 
 # Preset search seeds
-PresetSearch.create!(name: 'Test Search', search_terms: ["Test Study"],
+PresetSearch.create!(name: 'Test Search', search_terms: ["Testing Study"],
                      facet_filters: ['species:NCBITaxon_9606', 'disease:MONDO_0000001'], accession_whitelist: %w(SCP1))
+FeatureFlag.create!(name: 'faceted_search')
+
+# seed BQ for search controller test
+puts "Directly seeding BigQuery w/ synthetic data"
+bq_seeds = File.open(Rails.root.join('db', 'seed', 'bq_seeds.json'))
+bq_data = JSON.parse bq_seeds.read
+bq_data.each do |entry|
+  entry['CellID'] = SecureRandom.uuid
+  entry['study_accession'] = study.accession
+  entry['file_id'] = metadata_file.id.to_s
+end
+tmp_file = File.new('tmp_bq_seeds.json', 'w+')
+tmp_file.write bq_data.map(&:to_json).join("\n")
+table = ApplicationController.big_query_client.dataset(CellMetadatum::BIGQUERY_DATASET).table(CellMetadatum::BIGQUERY_TABLE)
+job = table.load(tmp_file, write: 'append', format: :json)
```

Comment:
given that this is 'append', will we ever delete cells from the test BQ instance?  it seems like this might as well write the table fresh, to mirror how we clean out the test mongoDB with each run

Reply:
I agree, but for some reason when I didn't include the "append" mode, the job fails every time.  I'll try it without again but I couldn't get it to work correctly without this.

Reply:
makes sense.  The data is scoped by study/file so it shouldn't be a problem -- definitely something we can save for later